src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -31,39 +56,43 @@ public String todayDate(DateTimePart datePart) {
      *                   - 3-й элемент массива - день (число);
      */
     public Optional<LocalDate> getDate(Integer[] dateParams) {
```

Comment:
Maybe it needs some check for ArrayIndexBound, what if dateParams.length will be one or zero? Omg I have found an error in my code because of this check :)